website/docs/cloud-docs/users-teams-organizations/permissions.mdx

```diff
@@ -235,6 +234,17 @@ Allows members to manage the set of [VCS providers](/cloud-docs/vcs) and [SSH ke
 
 Allows members to publish and delete providers, modules, or both providers and modules in the organization's [private registry](/cloud-docs/registry). These permissions are otherwise only available to organization owners.
 
+### Manage Membership
+
+Allows members to invite users to the organization, remove users from the organization, and add or remove users from teams within the organization.
+
+This permission grants the ability to view the list of users within the organization, and to view the organization access of other visible teams. It does not permit the creation of teams, the ability to modify the settings of existing teams, or the ability to view secret teams.
+
+In order to modify the membership of a team, a user with Manage Membership permissions must have visibility into the team (i.e. the team must be ["Visible"](/cloud-docs/users-teams-organizations/teams#team-visibility), or the user must be on the team).
+In order to remove a user from the organization, the holder of this permission must have visibility into all of the teams which the user is a member of.
+
+~> This permission is intended to allow owners of large organizations to delegate membership management to another trusted team, and should be granted to only teams of trusted users. **Assign with caution:** Users with this permission are able to add themselves to any visible team, and inherit the permissions of any visible team.
```

Comment:
Quick suggestion: Moving the "only" here earlier in the sentence flows a bit better/adds a little more emphasis on the "only" to me -- what do you think?
```suggestion
~> This permission is intended to allow owners of large organizations to delegate membership management to another trusted team, and should only be granted to teams of trusted users. **Assign with caution:** Users with this permission are able to add themselves to any visible team, and inherit the permissions of any visible team.
```